<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyGenius V.1.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="greenButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>120</width>
      <height>81</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
width: 98%;
padding: 10px;
border-color:black;
border-radius: 15px;

border:2px solid lime;
background-color: lime;
color: white;
font-family: 'Orbitron', sans-serif;
}
QPushButton:hover{
	
	background-color: rgb(143, 240, 164);
    color: rgb(143, 240, 164);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="redButton">
    <property name="geometry">
     <rect>
      <x>419</x>
      <y>100</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
width: 98%;
padding: 10px;
border-color:black;
border-radius: 15px;

border:2px solid red;
background-color: red;
color: white;
font-family: 'Orbitron', sans-serif;
}
QPushButton:hover{
	background-color: rgb(246, 97, 81);
    color:black;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="yellowButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
width: 98%;
padding: 10px;
border-color:black;
border-radius: 15px;

border:2px solid yellow;
background-color: yellow;
color: white;
font-family: 'Orbitron', sans-serif;
}
QPushButton:hover{
	background-color:#ADFF2F;
    color:black;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="blueButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
width: 98%;
padding: 10px;
border-color:black;
border-radius: 15px;

border:2px solid navy;
background-color: navy;
color: white;
font-family: 'Orbitron', sans-serif;
}
QPushButton:hover{
	
	background-color: rgb(28, 113, 216);
    color:rgb(28, 113, 216);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
width: 98%;
padding: 10px;
border-color:black;
border-top-left-radius: 15px;
border-bottom-right-radius: 12px;
border:2px solid black;
background-color: #323232;
color: white;
font-family: 'Orbitron', sans-serif;
text-spacing:4px;
}
QPushButton:hover{
	background-color:#ADFF2F;
    color:black;
}
</string>
    </property>
    <property name="text">
     <string>INICIAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="timerLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 98%;
padding: 10px;
border-color:black;
border-top-left-radius: 15px;
border-bottom-right-radius: 12px;
border:2px solid black;
background-color: #323232;
color: orange;
font-family: 'Orbitron', sans-serif;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restartButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>380</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
width: 98%;
padding: 10px;
border-color:black;
border-top-left-radius: 15px;
border-bottom-right-radius: 12px;
border:2px solid black;
background-color: #323232;
color: white;
font-family: 'Orbitron', sans-serif;
text-spacing:4px;
}
QPushButton:hover{
	background-color:#ADFF2F;
    color:black;
}
</string>
    </property>
    <property name="text">
     <string>REINICIAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>11</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../gen.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:12px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Scripts_python/piootion_2/piootion.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
